Add a section about Slack to the participation guide (#21)
open-feature/open-feature.github.io: content/home/participate.md

This is a public listing for those individual and company contributors
who declared their interest in the project.

## Slack

We plan to create the official Slack workspace later so that multiple workstreams and channels could be established there.

> For now, we have an `$openfeature` Slack channel on the CNCF Slack Workspace.

Reviewer: ```suggestion
For now, we have an `openfeature` Slack channel on the CNCF Slack Workspace.
```

You can join this Slack using [this link](https://slack.cncf.io/).

## Project meetings

Stay tuned for the announcements!

> **[Sign-up for news](https://bit.ly/openfeature-signup)** to get invited the meetings.

Reviewer: ```suggestion
**[Sign-up](https://bit.ly/openfeature-signup)** to get invited the meetings.
```

> **[Sign-up for news](https://bit.ly/openfeature-signup)** to get invite to the time poll and the meetings

Reviewer: This sentence is a bit confusing. Could you please rephrase it?

Author: ```suggestion
**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invited the meetings.
```

> until the official channel is set up.

Reviewer: This section can be removed.
